hwilib/devices/ledger.py

```diff
@@ -127,15 +126,16 @@ def __init__(self, path, password='', expert=False):
     # Must return a dict with the xpub
```

Comment:
not true anymore

Reply:
I've removed all of these comments since the docstrings in `HardwareWalletClient` supersede them.